data/fields/healthcare/speciality.json

```diff
@@ -1,5 +1,94 @@
 {
     "key": "healthcare:speciality",
     "type": "semiCombo",
-    "label": "Specialties"
+    "label": "Specialties",
+    "strings": {
+        "options": {
+            "abortion": "Abortion",
+            "acupuncture": "Acupuncture",
+            "allergology": "Allergy & Asthma",
+            "anaesthetics": "Anesthesiology",
+            "angiology": "Angiology",
+            "anthroposophical": "Anthroposophic Medicine",
+            "applied_kinesiology": "Applied Kinesiology",
+            "aromatherapy": "Aromatherapy",
+            "ayurveda": "Ayurveda",
+            "behavior": "Behavior Therapy",
+            "biochemistry": "Biochemistry",
+            "biology": "Biology",
+            "blood_check": "Blood Test",
+            "body": "Body Therapy",
+            "cardiology": "Cardiology",
+            "cardiothoracic_surgery": "Cardiothoracic Surgery",
+            "child_psychiatry": "Child and Adolescent Psychiatry",
+            "chiropractic": "Chiropractic",
+            "clinical_pathology": "Clinical Pathology",
+            "community": "Community and Public health",
+            "depth": "Depth Psychology",
+            "dermatology": "Dermatology",
+            "dermatovenereology": "Dermatovenereology",
+            "diabetology": "Diabetology",
+            "diagnostic_radiology": "Diagnostic Radiology",
+            "emergency": "Emergency Medicine",
+            "endocrinology": "Endocrinology",
+            "fertility": "Fertility",
+            "gastroenterology": "Gastroenterology",
+            "general": "General Practitioner",
```

Comment:
Should we make this “General Practice” for consistency with the other fields, which name fields of medicine instead of occupations?